content/en/docs/components/pipelines/sdk/connect-api.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "Connecting to API"
+description = "How to connect to Kubeflow Pipelines API using the SDK and configure it using environment variables"
+weight = 80
++++
+
+This guide introduces:
+
+* how to connect to your Kubeflow Pipelines API using [the Kubeflow Pipelines SDK](/docs/components/pipelines/sdk/sdk-overview/) in general.
+* how to [configure the SDK client using environment variables](#configure-sdk-client-by-environment-variables).
+
+## Before you begin
+
+* You need a Kubeflow Pipelines deployment using one of the [installation options](/docs/components/pipelines/installation/overview/).
+* [Install the Kubeflow Pipelines SDK](/docs/components/pipelines/sdk/install-sdk/).
+
+## How Kubeflow Pipelines SDK connects to API?
```

Comment:
NIT: Section title is usually not a question. We can use something like `Connect to API endpoint from SDK`.

Reply:
Good point! Will update